src/test/java/com/hotels/mutantswarm/mutate/ParserMutatorStoreTest.java

```diff
@@ -170,5 +193,66 @@ public void checkInvalidMutator() {
     List<Mutator> mutators = database.getMutatorsFor(node1);
     assertThat(mutators.size(), is(0));
   }
+  
+  @Test
+  public void checkMutatorsForPlus() {
+    when(node1.getType()).thenReturn(Vocabulary.INSTANCE.getId("PLUS"));
+    List<Mutator> mutators = database.getMutatorsFor(node1);
+    assertThat(mutators.size(), is(1));
+    assertThat(mutators.get(0).getDescription(), is("Op PLUS → MINUS '-'"));
+  }
+  
+  @Test
+  public void checkMutatorsForMinus() {
+    when(node1.getType()).thenReturn(Vocabulary.INSTANCE.getId("MINUS"));
+    List<Mutator> mutators = database.getMutatorsFor(node1);
+    assertThat(mutators.size(), is(1));
+    assertThat(mutators.get(0).getDescription(), is("Op MINUS → PLUS '+'"));
+  }
+  
+  @Test
+  public void checkMutatorsForStar() {
+    when(node1.getType()).thenReturn(Vocabulary.INSTANCE.getId("STAR"));
+    List<Mutator> mutators = database.getMutatorsFor(node1);
+    assertThat(mutators.size(), is(1));
+    assertThat(mutators.get(0).getDescription(), is("Op MUL → DIV '/'"));
+  }
+  
+  @Test
+  public void checkMutatorsForDivision() {
+    when(node1.getType()).thenReturn(Vocabulary.INSTANCE.getId("DIV"));
+    List<Mutator> mutators = database.getMutatorsFor(node1);
+    assertThat(mutators.size(), is(1));
+    assertThat(mutators.get(0).getDescription(), is("Op DIV → MUL '*'"));
+  }
+  
+  @Test
+  public void checkMutatorsForFalse() {
+    when(node1.getType()).thenReturn(Vocabulary.INSTANCE.getId("KW_FALSE"));
+    List<Mutator> mutators = database.getMutatorsFor(node1);
+    assertThat(mutators.size(), is(1));
+    assertThat(mutators.get(0).getDescription(), is("BooleanLiteral FALSE → TRUE 'true'"));
+  }
+  
+  @Test
+  public void checkMutatorsForTrue() {
+    when(node1.getType()).thenReturn(Vocabulary.INSTANCE.getId("KW_TRUE"));
+    List<Mutator> mutators = database.getMutatorsFor(node1);
+    assertThat(mutators.size(), is(1));
+    assertThat(mutators.get(0).getDescription(), is("BooleanLiteral TRUE → FALSE 'false'"));
+  }
+  
+  @Test
+  public void checkMutatorsHasBooleanChild() {
+    ArrayList<Node> children = new ArrayList<Node>();
```

Comment:
```suggestion
    List<Node> children = new ArrayList<Node>();
```
Always keep the left hand side as generic as possible (i.e. using an interface rather than a concrete class type) so that you can change the type on the right hand side without having to make any other changes as long as it implements the same interface.

Reply:
In line 250, the `.getChildren()` method must return a type `ArrayList<Node>`, so I can leave it like this or change this to `List<Node>` and then change line 250 to `when(node1.getChildren()).thenReturn((ArrayList<Node>) children);`

Reply:
Ah, OK, I see that, once again, the Hive developers have let us down by following the bad practice of using concrete types rather than interfaces in their method signatures. So just leave it like this as that's what has to be passed in.